my-app/src/Dashboard.js

```diff
@@ -669,7 +669,7 @@ class Dashboard extends Component {
 						// Sets the resolved state's message
 						//console.log("Debugging2: "+ i + " =  "+ currObject[i] );
 						resolve("Unlink of UserUi: " + currObject[i] + " successful");
-						boardNamesRef = firebase.database().ref('listOfBoards/'+uid+'/userId+/'+i);
```

Comment:
This was the "bug" I was talking about earlier Jeremy & Rick. Pretty sure this easy fix is acceptable and will make everything works as intended